src/elements/TabbedContainer/Tab.js

```diff
@@ -0,0 +1,31 @@
+import './TabbedContainer.less';
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Div from '../Div/Div';
+
+export default class Tab extends Component {
```

Comment:
Since tabs are stateless components, this would be best defined as a stateless functional component:

```
const Tab = props => {
    ....
};

...

export default Tab;
```

Reply:
Let's see if there's an eslint option for this (to yell at you if it doesn't see the `this` keyword)